bundles/org.openhab.binding.miele/src/main/java/org/openhab/binding/miele/internal/handler/MieleBridgeHandler.java

```diff
@@ -163,6 +168,14 @@ public String getConnectionType() {
             return connectionType.getAsString();
         }
 
+        public String getConnectionBaudRate() {
+            JsonElement baudRate = Properties.get("connection.baud.rate");
```

Comment:
Please add `@Nullable` to the method return. 

And if not too much work, switch the class to non null by default.

Reply:
> Please add `@Nullable` to the method return. 

Currently that would be redundant as class is null by default.

> And if not too much work, switch the class to non null by default. 

The class is for JSON deserialization, so I'm not sure it makes much sense? It would require a massive amount of @Nullable decorations except for a few methods.

Reply:
Ok, I see, it is in class HomeDevice, not in class MieleBridgeHandler.

I guess it is a very old binding ?

Reply:
> I guess it is a very old binding ?

Yes, since 1.x I think. I've refactored it a lot recently, but still much to do. For example extracting HomeDevice. :-)